_layouts/page.html

```diff
@@ -4,18 +4,20 @@
 
 {% include hero.html %}
 {% assign lang = page.lang | default: site.lang | default: "en" %}
-<div class="documentation">
+<section class="documentation">
   {% include sidebar.html %}
   <div class="main-pane">
-    <div class="markdown">{{ content }}</div>
-    {% comment %}
-        Use the github edit link until netlify-cms support easy deep link.
-        See: https://github.com/netlify/netlify-cms/issues/697
-    {% endcomment %}
-    <p class="text-center">
-        <a class="edit-button" href="https://github.com/{{ site.github_repository }}/edit/master/{{ page.path }}">
-            ✎ {{ site.i18n[lang].edit }}
-        </a>
-    </p>
+    <div class="markdown">
+      {{ content }}
+      {% comment %}
+          Use the github edit link until netlify-cms support easy deep link.
+          See: https://github.com/netlify/netlify-cms/issues/697
+      {% endcomment %}
+      <p class="text-center">
```

Comment:
Why putting this in the `div.markdown` ? This is not some markdown and so, `p` and `a` styles will apply on this button as a side effect, no ?

Reply:
There's a width on the `.markdown` container, if I leave the button outside it will be centered regarding the main pane and will not be aligned regarding the text.